teamwork/templates/dashboard.html

```diff
@@ -2,19 +2,30 @@
 {% load extra_tags %}
 
 {% block content %}
+<div id="page-wrapper">
+  <div class="container">
+    <h3>Next Event</h3>
+    
+  </div>
 
-<h1>
-    Dashboard
-</h1>
-<p>"Hello world, and mars"</p>
+  <div class="container">
+    <h3>Ongoing Projects</h3>
+    {% for project in projects %}
+    <div class="jumbotron">
+      <h2><a href="{% url 'project_details' user.username project.name|urlencode_plus %}">{{ project.name }}</a></h2>
+      {% for event in project.event_set.all|dictsort:'date' %}
+      {% if forloop.first %}

```

Comment:
![capture](https://cloud.githubusercontent.com/assets/22631687/19648463/f32aaba4-9a0a-11e6-9553-25e6f5e0530c.JPG)

It's like this now